internal/cluster/distribution/eks/eksworkflow/activity_update_addon.go

```diff
@@ -49,13 +50,15 @@ type UpdateAddonActivityInput struct {
 	ClusterName       string
 	KubernetesVersion string
 
-	AddonName string
+	AddonName                 string
+	UpgradeToNextMinorVersion bool
```

Comment:
**Recommended**: i think we **SHOULD** rename the field `UpgradeToNextMinorVersion` to `UpgradeAtMostOneMinorVersion`, because we are using it (the true value) in the  context where we also allow patch upgrades to happen with no minor change which makes it slightly misleading as no minor update happens then and the original name instructs to upgrade exactly one minor version. Renaming would decrease cognitive load of remembering the exact semantics.

```suggestion
	UpgradeAtMostOneMinorVersion bool
```

Reply:
Fixed in https://github.com/banzaicloud/pipeline/pull/3563/commits/22da7696190597e681af96bcbc1fc21ee85163dc.